test/test_mpi_p4est_2d.jl

```diff
@@ -69,7 +69,10 @@ const EXAMPLES_DIR = pkgdir(Trixi, "examples", "p4est_2d_dgsem")
     @trixi_testset "elixir_advection_restart.jl" begin
         @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_restart.jl"),
                             l2=[4.507575525876275e-6],
-                            linf=[6.21489667023134e-5])
+                            linf=[6.21489667023134e-5],
+                            # With the default `maxiters = 1` in coverage tests,
+                            # there would be no time steps after the restart.
+                            coverage_override = (maxiters = 100_000,))
```

Comment:
> Update: I can reproduce it now. When running the tests with coverage, `maxiters=1` is inserted. This is why I didn't have this problem locally when running without coverage.

I came up with two ideas how to solve this problem.
1. Ignore if `maxiters` is not found in `find_assignment`. But then we don't catch errors where a user wants to pass `maxiters`, but it doesn't have any effect because it's not found.
2. Add `maxiters = 100_000` (which is the default that is automatically added by `trixi_include` anyway) to all restart elixirs, and add a coverage override to keep it at this value and not set it to `maxiters = 1`. Otherwise, we wouldn't do any steps after the restart in coverage tests.

I chose the second option.

Reply:
👍